src/sentry/tasks/files.py

```diff
@@ -5,9 +5,10 @@
 
 from sentry.silo.base import SiloMode
 from sentry.tasks.base import instrumented_task, retry
-from sentry.tasks.deletion.scheduled import MAX_RETRIES
 from sentry.utils.db import atomic_transaction
 
+MAX_RETRIES = 5
```

Comment:
Importing this from the new location was creating import cycles. This was the simplest solution to resolve the cycles.